src/endpoints/v1.es6.js

```diff
@@ -115,9 +119,9 @@ function returnGETPromise (options, formatBody, log) {
   });
 }
 
-function bind(obj, context) {
-  for (var p in obj) {
-    if(obj.hasOwnProperty(p) && typeof obj[p] == 'function' ) {
+function bindAll(obj, context) {
+  for (let p in obj) {
```

Comment:
:no_bell: (sort of) @curioussavage (because you're working on styleguide)

Styleguide recommends using `.forEach` (and other higher order functions) for iterating. I don't know if we do that consistently. Should we update that rule to note when it's okay to not do so?